config/env_config.py

```diff
@@ -10,6 +10,12 @@ def __init__(self):
         self.test_webhook_url = get_secret(
             "boost-bot/test-discord-url", "TEST_WEBHOOK_URL"
         )
+        self.test_error_url = get_secret(
+            "boost-bot/test-error-discord-url", "TEST_WEBHOOK_URL"
```

Comment:
should this line be here? that secret only has DISCORD_WEBHOOK_URL as a key

Reply:
`boost-bot/test-error-discord-url` is a dummy test webhook for what I was locally testing, just converted to the `get_secret` so I don't forget to change.

Reply:
the true error is the secret should be `"boost-bot/prod-error-discord-url"` :/